scripts/find-updates.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+set -euo pipefail
+# set -x
+
+# referenced from https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c
+function get_latest_release() {
+  version=$(curl --silent "https://api.github.com/repos/$1/releases/latest" |
```

Comment:
I think using `jq` would be nicer. Like in https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c#gistcomment-2574561